executor/index_merge_reader.go

```diff
@@ -648,6 +654,18 @@ func (w *indexMergeTableScanWorker) pickAndExecTask(ctx context.Context) {
 	}
 }
 
+func (w *indexMergeTableScanWorker) handlePickAndExecTaskPanic(ctx context.Context, task *lookupTableTask) func(r interface{}) {
```

Comment:
Can we pass the `task.doneCh` instead of `task` here?

Reply:
task had not init and task.doneCh is nil.